atlas_admin/helpers.py

```diff
@@ -345,3 +345,71 @@ def uniqueTupleList(li):
 
 def lowerList(mixedList):
     return [x.lower() for x in mixedList]
+
+
+def statBreakdown():
+    """
+    Returns a summary breakdown of statistics
+    """
+
+    q = get_internal('statistics')
+    res = q[0] if len(q) > 0 else {}
+    totalItems = res.get('_meta', None).get('total', None)
+
+    summary = {}
+
+    if totalItems:
+
+        # If more items exist than initial request, reset max_results to total to get a full export
+        if totalItems > res.get('_meta', None).get('max_results', None):
+            # Copy the existing arguments on the request object
+            setArgs = request.args.copy()
+            # Set our new header
+            setArgs['max_results'] = totalItems
+            request.args = setArgs
+            results = get_internal('statistics')[0]['_items']
+        else:
+            results = res.get('_items', None)
+
+        themeCount = Counter()
+        for res in results:
+            if 'variable_theme_default' in res:
+                themeCount[res['variable_theme_default']] += 1
+        themeList = dict(themeCount)
+        sortedThemeList = sorted(themeList.items(), key=lambda x: x[1])
+        summary['variable_theme_default'] = OrderedDict(sortedThemeList)
+    else:
+        summary = None
+
+    return summary
+
+
+def sitesByStat(themeName=None):
```

Comment:
Let's see if there is a way to pass in both the key and value that you are searching for. That would mean that it would work for any stat without adjusting the function. We want to avoid having a function with dozens of optional arguments.